### PR TITLE
update /pricing & /self-hosted.

### LINKED ISSUE #1284: Update /pricing
**Major changes:**
- the self-hosted plans are becoming more present
- a feature table is added to compare plans
- the FAQ section is extended and updated
- a trusted by section is added
- tooltips are added to provide more information on certain features
- the responsive design is improved

**Mockups**
Note: there are some comments in the figma file to clear questions about the content and implementation
Pricing -new
Self hosted -new
[mockups](https://www.figma.com/file/K0VV8lc3pmlMVModYM70nj/Website?node-id=2828%3A1288)

### PR DESCRIPTION
**Pull request by nisarhassan12**
Fixes #1284 #176 

### Preview URLs:

- Self Hosted: https://deploy-preview-1329--gitpod-kumquat.netlify.app/self-hosted
- Pricing: https://deploy-preview-1329--gitpod-kumquat.netlify.app/pricing 

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1329"></a>

### Remaing todos:

- [x] add the features table on `/pricing`. 

### Will be done in a follow up:

- [ ] update the toggle design https://github.com/gitpod-io/website/issues/1387.
- [ ] Making the tables consistent https://github.com/gitpod-io/website/issues/1384.
- [x] add support for links inside the tooltips.
- [ ] add the popovers to the mobile versions of the feature tables https://github.com/gitpod-io/website/issues/1388.

### CODE REVIEW

File: src/routes/pricing.svelte [hunk 1]
```diff
@@ -19,7 +20,12 @@
...
   }}
 />
-<PlansAndPricing {trackingContext} />
-<OtherPlans {trackingContext} {otherPlans} />
+<PlansAndPricing {trackingContext} {pricingPlans}>
+  <p slot="note" class="mt-micro text-center">
+    Can’t find the answer here? <a href="/contact/sales">Contact sales</a>
+  </p>
+</PlansAndPricing>
+<UsedBy title="Used by 400k+ developers" />
```
Comment (nisarhassan12) [inline src/routes/pricing.svelte:28]: This needs to be changed to 500k+

**Comment by nisarhassan12** [comment]
The Self-hosted page is ready to be reviewed there is still some work remaining for the pricing page though.

File: src/contents/self-hosted.ts [hunk 2]
```diff
@@ -34,15 +35,149 @@ export const pricingPlans: Pricing[] = [
...
     ],
     btnText: "Register and Install now",
     btnHref: "/enterprise-license",
     spiced: true,
     trackingName: "professional",
   },
 ];
+
+export const freePlanTableData: TableData = {
+  title: "Basic",
```
Comment (metcalfc) [inline src/contents/self-hosted.ts:52]: Free

File: src/contents/pricing.ts [hunk 3]
```diff
@@ -1,15 +1,23 @@
 import type { Pricing } from "../types/pricing.type";
 import { isEurope } from "../utils/helpers";
+import type { Card } from "../types/card.type";
 
 export const pricingPlans: Pricing[] = [
   {
     title: "Basic",
```
Comment (metcalfc) [inline src/contents/pricing.ts:7]: Basic has been renamed to Free.

File: src/contents/self-hosted.ts [hunk 4]
```diff
@@ -34,15 +35,149 @@ export const pricingPlans: Pricing[] = [
...
+      availibility: true,
+    },
+  ],
+};
+
+export const professionalPlanTableData: TableData = {
+  title: "Professional",
+  subtitle: `${isEurope() ? "€29" : "$35"} per user/month`,
+  link: {
+    text: "Contact Sales",
```
Comment (metcalfc) [inline src/contents/self-hosted.ts:122]: Contact Sales should be "Register and Install Now"

**Review by metcalfc — CHANGES_REQUESTED** [review]
Two of the FAQ entries also need updating.

```Can I create a team plan?

Of course! You can use Gitpod Self-Hosted Community on your own infrastructure for free for unlimited users. If you'd like to try additional features like unlimited prebuilds, shared workspaces, snapshots and an admin dashboard, the Free tier is available without a license for the first 10 users. You can upgrade to to the Professional Plan at any time. Simply update your existing instance with a license key. Currently, Self-hosted Gitpod licenses can currently only be purchased through contacting sales . Please [request a license ](/enterprise-license)key or get in touch if you have any questions.
```

```How can I install Self-Hosted?


You can either install Gitpod Self-Hosted on [Google GKE](/docs/self-hosted/latest/installation/on-gke), [Amazon EKS](/docs/self-hosted/latest/installation/on-eks), [Azure AKS](/docs/self-hosted/latest/installation/on-aks) or [K3s](/docs/self-hosted/latest/installation/on-k3s). 
```

**Comment by nisarhassan12** [comment]
Thanks, Chad. I have made the requested changes in https://github.com/gitpod-io/website/pull/1329/commits/ce0e4aa805bf476ea743e104249c9516a30f0089. 

The Pricing page is now also ready to be reviewed.

File: src/components/self-hosted/faqs.svelte [hunk 5]
```diff
@@ -53,9 +53,15 @@
...
+        <a href="/docs/self-hosted/latest/installation/on-amazon-eks"
+          >Amazon EKS</a
+        >,
+        <a href="/docs/self-hosted/latest/installation/on-microsoft-aks"
+          >Azure AKS</a
+        >
+        or
+        <a href="/docs/self-hosted/latest/installation/on-kubernetes">K3s</a>.
+        See
         <a href="/docs/self-hosted/latest">Self-Hosted Docs</a> for more information.
```
Comment (nisarhassan12) [inline src/components/self-hosted/faqs.svelte:65]: @metcalfc Should I remove lines 64 and 65 ? Thanks

**Comment by metcalfc** [comment]
I can't find it here but one of the saas pages has 0$ instead of $0 once that's fixed I think we are good to go

**Review by mikenikles — CHANGES_REQUESTED** [review]
## `/pricing`

### Used by logos

The logos have various sizes, with Amazon being super large. Looking at the design spec, they should all be more aligned.

<details>
  <summary>Click to view screenshot</summary>
  
</details>

### Get started

The design shows "Get started for free" whereas this PR only has "Get Started". It'd be good to match the marketing wording from the design.

## `/self-hosted`

### Used by logos

This section is missing.

**Comment by nisarhassan12** [comment]
> The logos have various sizes, with Amazon being super large. Looking at the design spec, they should all be more aligned.

Thanks, @mikenikles they follow the specs see the below screenshot Not sure why that is the case for you maybe for some reason some CSS didn't load for you. I have tested on Firefox, Chrome, Brave and Safari on Desktop and on  Firefox, Chrome, Brave on my phone and was not able to reproduce this.

<details>
<summary>Screenshot</summary>

</details>

> This section is missing.

This Pr initially had that I removed it in https://github.com/gitpod-io/website/pull/1329/commits/adfde91f7346d79b1cd65761e01bb7d928f9d8b0 the reason why I did this can be found in this Slack [thread](https://gitpod.slack.com/archives/C01KGM9ETU6/p1640159139253300?thread_ts=1640140350.251400&cid=C01KGM9ETU6) Sorry I should have mentioned that in the PR. 

> The design shows "Get started for free" whereas this PR only has "Get Started". It'd be good to match the marketing wording from the design.

✅ Done. Sorry I completely missed that.

**Review by mikenikles — APPROVED** [review]
Thanks Nisar for the explanations. LGTM :+1:

**Pull request merged**